tutor/src/components/notes/sidebar-buttons.jsx

```diff
@@ -69,8 +69,17 @@ class SidebarButtons extends React.Component {
     windowImpl: PropTypes.object,
   }
 
-
   @observable containerTop = null;
+  // Add a resize dummy listener to listen again and get the new container rect specs
+  @observable fakeRender = false;
+
+  componentDidMount() {
+    window.addEventListener('resize', () => this.fakeRender = !this.fakeRender);
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('resize', () => this.fakeRender = !this.fakeRender);
+  }
 
```

Comment:
instead of using manual listeners use window size model, it auto listens and unlistens and is what the responsive component is built on. `https://github.com/openstax/tutor-js/blob/main/tutor/src/models/window-size.js`. If you rewrite `getRangeRect` to accept the window size from that it should re-render whenever the window size changes.  It also debounces so you don't re-render a 1000 times on any little resize